workflows/wf_titan_fasta.wdl

```diff
@@ -15,25 +15,25 @@ workflow titan_fasta {
 
   input {
     String  samplename
-    File    assembly_fasta
-    String  seq_method  
+    File    fasta
```

Comment:
Only suggestion I have is that we keep the assembly_fasta vocabulary to stay consistent with the other workflow outputs; also helpful when training and onboarding labs as we can introduce the idea of selecting an _assembly_ file in _fasta_ format to the _assembly_fasta_ input attribute

Reply:
Ok, I'll have to change on the cli side. It didn't like when I used `assembly_fasta`